ecosystem/sep-0008.md

```diff
@@ -225,17 +225,56 @@ Name | Type | Description
 `status` | string | `"action_required"`
 `message` | string | A human readable string containing information regarding the action required.
 `action_url` | string | A URL that allows the user to complete the actions required to have the transaction approved.
+`action_method` | string | (Optional) `GET` or `POST`, indicating the type of request that should be made to the `action_url`. If not provided, `GET` is assumed.
```

Comment:
nit: I think in other places of this doc, we use `(optional)` instead of `(Optional)`. Maybe it's worth making it consistent.

Reply:
https://github.com/DSTOQ/stellar-protocol/pull/4